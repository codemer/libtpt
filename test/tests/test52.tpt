@repeat("x", 10)
@repeat("x", 0)
@repeat("x", -1)
@repeat("NULL, ", 5)
