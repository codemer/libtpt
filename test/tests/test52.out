xxxxxxxxxx


NULL, NULL, NULL, NULL, NULL, 
